feature-toggle-api/templates/ingress.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.ingressHost }}
```

Comment:
the dns isn't being registered here if its a sub chart, if you want to make ingress an option you may want to add a register dns flag that will add it, see the chart-ccd for how to do it